Gruntfile.js

```diff
@@ -3,12 +3,23 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('package.json'),
     sass: {
       options: {
+        includePaths: [
+          "node_modules/bourbon/app/assets/stylesheets",
```

Comment:
Include paths let us use NPM for lib dependency management. This is how it is done.